pkg/aws/aws.go

```diff
@@ -122,16 +106,23 @@ func NewClientFromFileCredentials(dir string, region string) (*SdkClient, error)
 	}
 	accessKeyID := strings.TrimRight(string(accessKeyBytes), "\n")
 	secretKeyID := strings.TrimRight(string(secretKeyBytes), "\n")
+	// XXX: Does an empty session token work?
```

Comment:
**chore:** any action to be taken for this comment? What does the `XXX` stand for?

Reply:
It seems that they only renamed what the parameter was called so I think there is no action: `XXX` just gets highlighted in my editor there are some functions to show me all occurrences in the project.